interface/super/edit_list.php

```diff
@@ -955,9 +955,13 @@ function mysubmit() {
                   }else if ( $list_id == 'immunization_observation' ) {
                         xl('LOINC Code','e'); 
                   
-      } else {
+                  }else if ( $list_id == 'page_validation' ) {
+                              xl('Page Validation - ','e');
```

Comment:
This is a very odd translated constant(note constants should not have trailing/leading spaces or using symbols), better to do:
``````
xl('Page Validation');